SequenceAnalysis/src/org/labkey/sequenceanalysis/SequenceAnalysisController.java

```diff
@@ -434,8 +435,21 @@ public static class DownloadTempImageAction extends ExportAction<TempImageAction
         @Override
         public void export(TempImageAction form, HttpServletResponse response, BindException errors) throws Exception
```

Comment:
@bbimber - Can you please provide a usage scenario for this action. Thanks!

Reply:
are you asking for a way to test it?

Reply:
So I believe both DownloadTempImageAction and ConvertTextToFileAction are really old code that I dont see used anywhere. Rather than refactor them, those classes and the associated form classes could be removed.

Reply:
Well was wondering how that action gets called and how the values for directory and filename gets set.
But sounds good, if they are not being used, then I can remove them as part of this PR.